src/controllers/settings.controller.js

```diff
@@ -0,0 +1,34 @@
+import ResponseService from '../services/response.service';
+import UserService from '../services/user.service';
+/**
+ *
+ *
+ * @class SettingsController
+ */
+class SettingsController {
+  /**
+  * super_admin can assign a role to user by email
+  * @static
+  * @description PATCH /api/admin/reset-user-role
+  * @param {object} req request object
+  * @param {object} res response object
+  * @memberof UserRoleSettingsController
+  * @returns {object} ResponseService
+  */
+  static async changeUserRole(req, res) {
+    const { userEmail, userRole } = req.body;
+    const userExist = await UserService.findUserByProperty({ email: userEmail.trim() });
```

Comment:
Hey @higustave-ops, if you rename `userExist` to `findUser` or `findUserRole` or `user` could be better, because I can see down you are using this variable to find a role of a user. what do you think about that?

Reply:
@erickyvand , @adesege said that for a variable that will return a boolean, we use such kind of names to reflect to the return value. example: `isUserAdmin`: true or false `userExist`: true or false

Reply:
`findUserByProperty` does not return a boolean but an object if it can find a user or null if it can't. You should take @erickyvand feedback into consideration.

Reply:
@adesege yes. I now changed